modules/tiles/src/tileset/traversers/tileset-3d-traverser.js

```diff
@@ -13,7 +13,7 @@ export default class Tileset3DTraverser extends TilesetTraverser {
   }
 
   updateTileVisibility(tile, frameState) {
-    tile.updateVisibility(frameState);
+    super.updateTileVisibility(tile, frameState);
```

Comment:
Calling `super` (outside of constructors) is a bit of a yellow flag to me, it makes the code harder to follow. 

Ideally the logic in here should not be different between 3D tiles and I3S. We should be able to normalize the tiles in the respective loaders so that the same logic applies here.

I know this separation already existed, just indicating that this is making the code more complex than it should be.